packages/client/txGenerator.ts

```diff
@@ -0,0 +1,186 @@
+// Adapted from - https://github.com/Inphi/eip4844-interop/blob/master/blob_tx_generator/blob.js
+import { randomBytes } from 'crypto'
+import { Address } from '@ethereumjs/util'
+import { Client } from 'jayson/promise'
+import { BlobEIP4844Transaction } from '@ethereumjs/tx'
+import {
+  blobsToCommitments,
+  commitmentsToVersionedHashes,
+} from '@ethereumjs/tx/test/utils/blobHelpers'
+import { loadTrustedSetup, freeTrustedSetup } from 'c-kzg'
+const input = process.argv[2]
+const expected_kzgs = process.argv[3]
+
+const BYTES_PER_FIELD_ELEMENT = 32
+const FIELD_ELEMENTS_PER_BLOB = 4096
+const USEFUL_BYTES_PER_BLOB = 32 * FIELD_ELEMENTS_PER_BLOB
+const MAX_BLOBS_PER_TX = 2
```

Comment:
we also have MAX_BLOBS_PER_BLOCK just marking here to later validate if we are enforcing it while generating txs